news/2018-09-19-the-qat-gazette-september-2018.md

```diff
@@ -0,0 +1,68 @@
+
+## Opening
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+A new month brings new adventure, new challenges, and a new face to our humble blog!
+
+A lot has happened in the past month and as you might expect, you can find it all right here! From substantial changes to the way Beatmap Nominator applications are conducted, to ongoing developments within the Modding Discussions system and of course, some aforementioned new faces, keep reading to find out everything you need to know in and around the mapping and modding community.
+
+## Within the Beatmap Nominators
+by [Kibbleru](https://osu.ppy.sh/users/3193504)
+
+We had yet another exciting month within the Beatmap Nomination Group. The long-awaited new round of Beatmap Nominator applications was finally released in late September, but this time, we have made some significant changes to both the application and evaluation processes. Continue reading to find out more!
+
+### New Beatmap Nominator Applications
+
+As many of you may have followed, this round of Beatmap Nominator applications was finally opened to the public around late September. Any modder with sufficient modding activity within the past three months was free to apply to have their proficiency judged, in hopes of becoming a new and upcoming member of the Beatmap Nomination Group!
+
+This time around we changed things up a bit. First and foremost, we decided to bring back proficiency tests in order to judge each applicant’s knowledge and understanding of the Ranking Criteria. Another change we decided on was to the application cycle. Instead of releasing tests every 1.5 months, alternating between the osu! gamemode, and taiko / catch / mania, we have decided that from now on, Beatmap Nominator applications for *all* game modes will be released at the same time every 3 months. Best of luck!
+
+### Beatmap Nominator Retirements
+
+Fun times always have to come to an end eventually. This month, we give our farewells to two of our diligent Beatmap Nominators: [Plaudible](https://osu.ppy.sh/users/7149815) and [Jonawaga](https://osu.ppy.sh/users/3653035).
+
+On behalf of the community, we thank you for your hard work in bringing new content into the game, and we wish you good luck in your next steps in life!
+
+## Within the Quality Assurance Team
+by [Raiden](https://osu.ppy.sh/users/2239480)
+
+### Ongoing discussions
+
+Normally, every monthly issue of the QAT Gazette brings you exciting news and updates on a great diversity of fields within the osu! Mapping and modding community.
+
+Unfortunately, this month is somewhat sparse on the news front, since some important topics require further discussion and have not yet been resolved. However, we are writing this in order to inform you that we indeed have not simply forgotten about such matters!
+The topics still being debated include the following:
+
++ Discussion about the [management of hybrid mapsets](https://osu.ppy.sh/community/forums/topics/792131) 
```

Comment:
list should use `-`, just a style thing

Reply:
I assumed these gave an identical effect but fair enough, changed